app/src/main/java/com/example/weather/screen/home/WeatherFragment.kt

```diff
@@ -120,7 +188,23 @@ class WeatherFragment private constructor() : BaseFragment<FragmentWeatherBindin
 
     private fun onRefresh() {
         checkNetwork(activity)
-        mPresenter?.getWeather(mLatitude, mLongitude, mIsNetworkEnable)
+        val itemID = mWeatherList[mPosition].city + mWeatherList[mPosition].country
+        val currentID = mWeatherCurrent?.city + mWeatherCurrent?.country
+        val isCurrent = itemID == currentID
+        if (mIsNetworkEnable) {
+            mWeatherList[mPosition].latitude?.let {
+                mWeatherList[mPosition].longitude?.let { it1 ->
+                    mPresenter?.getWeather(
+                        it,
+                        it1,
```

Comment:
```suggestion
                        it,
                        it1,
```
không dùng mặc định như này khi có nested block,
phải đặt tên cho nó clear, sửa các chỗ khác nữa nhé!